### PR TITLE
Add get order book in huobi

### PR DESCRIPTION
**Pull request by cacofishadct**
1. Add get order book in huobi
2. some formatting change.
3. Add deposit address, withdraw api in huobi
4. fix huobi place limit order issue

### CODE REVIEW

File: xchange-huobi/src/main/java/org/knowm/xchange/huobi/dto/marketdata/HuobiDepth.java [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    return asks;
+  }
+
+  public Date getTs() {
+    return ts;
+  }
+
+  public void setTs(Date ts) {
+    this.ts = ts;
+  }
```
Comment (walec51) [inline xchange-huobi/src/main/java/org/knowm/xchange/huobi/dto/marketdata/HuobiDepth.java]: is this attribute needed?
I find it odd that the whole object is immutable except for this property

File: xchange-huobi/src/main/java/org/knowm/xchange/huobi/dto/marketdata/results/HuobiDepthResult.java [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+      @JsonProperty("status") String status,
+      @JsonProperty("ts") Date ts,
+      @JsonProperty("tick") HuobiDepth tick,
+      @JsonProperty("ch") String ch,
+      @JsonProperty("err-code") String errCode,
+      @JsonProperty("err-msg") String errMsg) {
+    super(status, errCode, errMsg, tick);
+    this.ts = ts;
+    this.ch = ch;
+    getResult().setTs(ts);
```
Comment (walec51) [inline xchange-huobi/src/main/java/org/knowm/xchange/huobi/dto/marketdata/results/HuobiDepthResult.java]: this is odd, these DTO object should represent the exchanges response and should not do any premature mapping nor have data coping logic

File: xchange-huobi/src/main/java/org/knowm/xchange/huobi/dto/marketdata/results/HuobiDepthResult.java [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+  }
+
+  public Date getTs() {
+    return ts;
+  }
+
+  public String getCh() {
+    return ch;
+  }
```
Comment (walec51) [inline xchange-huobi/src/main/java/org/knowm/xchange/huobi/dto/marketdata/results/HuobiDepthResult.java]: pls you document what are those two attributes and do we intent on using them somehow

File: xchange-huobi/src/main/java/org/knowm/xchange/huobi/dto/marketdata/HuobiDepth.java (hunk 1 above)
Comment (cacofishadct) [reply xchange-huobi/src/main/java/org/knowm/xchange/huobi/dto/marketdata/HuobiDepth.java]: Copied :). Removed now.

File: xchange-huobi/src/main/java/org/knowm/xchange/huobi/dto/marketdata/results/HuobiDepthResult.java (hunk 2 above)
Comment (cacofishadct) [reply xchange-huobi/src/main/java/org/knowm/xchange/huobi/dto/marketdata/results/HuobiDepthResult.java]: Also copied. I am not very sure about the usage. Removed them in next change.

File: xchange-huobi/src/main/java/org/knowm/xchange/huobi/dto/marketdata/results/HuobiDepthResult.java (hunk 3 above)
Comment (cacofishadct) [reply xchange-huobi/src/main/java/org/knowm/xchange/huobi/dto/marketdata/results/HuobiDepthResult.java]: Removed

**Pull request merged**